{"url": "https://api.github.com/repos/tensorflow/tensorflow/issues/comments/351893595", "html_url": "https://github.com/tensorflow/tensorflow/issues/15296#issuecomment-351893595", "issue_url": "https://api.github.com/repos/tensorflow/tensorflow/issues/15296", "id": 351893595, "node_id": "MDEyOklzc3VlQ29tbWVudDM1MTg5MzU5NQ==", "user": {"login": "facaiy", "id": 1112263, "node_id": "MDQ6VXNlcjExMTIyNjM=", "avatar_url": "https://avatars3.githubusercontent.com/u/1112263?v=4", "gravatar_id": "", "url": "https://api.github.com/users/facaiy", "html_url": "https://github.com/facaiy", "followers_url": "https://api.github.com/users/facaiy/followers", "following_url": "https://api.github.com/users/facaiy/following{/other_user}", "gists_url": "https://api.github.com/users/facaiy/gists{/gist_id}", "starred_url": "https://api.github.com/users/facaiy/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/facaiy/subscriptions", "organizations_url": "https://api.github.com/users/facaiy/orgs", "repos_url": "https://api.github.com/users/facaiy/repos", "events_url": "https://api.github.com/users/facaiy/events{/privacy}", "received_events_url": "https://api.github.com/users/facaiy/received_events", "type": "User", "site_admin": false}, "created_at": "2017-12-15T02:12:58Z", "updated_at": "2017-12-15T02:12:58Z", "author_association": "MEMBER", "body_html": "<p>Yes, I agree that the case might be rare, in fact the bug is fund by accident. Moreover, I don't think I know the role of <code>variable_scopes_count</code> clearly. So l'm glad to leave the issue open :-)</p>", "body_text": "Yes, I agree that the case might be rare, in fact the bug is fund by accident. Moreover, I don't think I know the role of variable_scopes_count clearly. So l'm glad to leave the issue open :-)", "body": "Yes, I agree that the case might be rare, in fact the bug is fund by accident. Moreover, I don't think I know the role of `variable_scopes_count` clearly. So l'm glad to leave the issue open :-)"}